### PR TITLE
Relax pubsub configuration assumptions

### PR DESCRIPTION
**Pull request by jessemyers**
- Queue URL may not be provided on the CLI
 - Handler mappings may not be configured as a property

### CODE REVIEW

File: microcosm_pubsub/daemon.py [hunk 1]
```diff
@@ -12,7 +12,7 @@ class ConsumerDaemon(Daemon):
 
     def make_arg_parser(self):
         parser = super(ConsumerDaemon, self).make_arg_parser()
-        parser.add_argument("--sqs-queue-url", required=True)
+        parser.add_argument("--sqs-queue-url")
```
Comment (jessemyers) [inline microcosm_pubsub/daemon.py]: I'm specifically changing my daemon to have a reasonable default for the queue URL; it makes local development configuration a lot simpler.

File: microcosm_pubsub/daemon.py [hunk 2]
```diff
@@ -23,27 +23,37 @@ def schema_mappings(self):
         """
         pass
 
-    @abstractproperty
     def handler_mappings(self):
         """
         Define the PubSub message media-type to handler mappings.
 
+        This function is an alternative to configuring `graph.sqs_message_handlers` as
```
Comment (jessemyers) [inline microcosm_pubsub/daemon.py]: Most of our daemons have stopped using this property; I'm not yet ready to remove it, but I am making it less required.

File: microcosm_pubsub/daemon.py (hunk 1 above)
Comment (amitbn) [inline microcosm_pubsub/daemon.py]: makes sense

**Comment by amitbn** [comment]
+1

**Pull request closed without merging**